content/blog/2023-09-19-iron-fish-wallet-server.mdx

```diff
@@ -0,0 +1,33 @@
+---
+author: iflabs
+title: Introducing the Iron Fish Wallet Server
+description: 
+image: /images/blog/introducing-wallet-server/hero-img.png
+tags: [ironfish]
+---
+
+Today we're excited to announce the initial release of the Iron Fish Wallet Server. This project aims to enable the building and running of a fully private light client without the need to run a local Iron Fish node. In this blog post, we'll provide an overview of the Wallet Server, its key features, and talk about how you can help us improve this project.
+
+## TL;DR:
+
```

Comment:
- A rate limited version of this server is running at walletserver.ironfish.network for testing